Frontend/ui-customer/src/Components/SettingSideBar/View.purs

```diff
@@ -74,17 +80,25 @@ settingsView state push =
-    , settingsMenuView {imageUrl : "ny_ic_fav,https://assets.juspay.in/nammayatri/images/user/ny_ic_fav.png", text : (getString FAVOURITES)  , tag : SETTINGS_FAVOURITES} push
+     settingsMenuView {imageUrl : "ic_past_rides,https://assets.juspay.in/nammayatri/images/user/ic_past_rides.png", text : (getString MY_RIDES), tag : SETTINGS_RIDES, iconUrl : ""} push
+    , settingsMenuView {imageUrl : "ic_fav,https://assets.juspay.in/nammayatri/images/user/ic_fav.png", text : (getString FAVOURITES)  , tag : SETTINGS_FAVOURITES, iconUrl : ""} push
     , if (isPreviousVersion (getValueToLocalStore VERSION_NAME) (if os == "IOS" then "1.2.5" else "1.2.1")) then textView []
-      else settingsMenuView {imageUrl : "ny_ic_emergency_contacts,https://assets.juspay.in/nammayatri/images/user/ny_ic_emergency_contacts.png" , text : (getString EMERGENCY_CONTACTS)  , tag : SETTINGS_EMERGENCY_CONTACTS} push
-    , settingsMenuView {imageUrl : "ny_ic_change_language,https://assets.juspay.in/nammayatri/images/user/ny_ic_change_language.png", text : (getString LANGUAGE), tag : SETTINGS_LANGUAGE} push
-    , settingsMenuView {imageUrl : "ny_ic_help,https://assets.juspay.in/nammayatri/images/user/ny_ic_help.png", text : (getString HELP_AND_SUPPORT), tag : SETTINGS_HELP} push
-    , settingsMenuView {imageUrl : "ny_ic_share,https://assets.juspay.in/nammayatri/images/user/ny_ic_share.png", text : (getString SHARE_APP), tag : SETTINGS_SHARE_APP} push
-    , settingsMenuView {imageUrl : "ny_ic_info,https://assets.juspay.in/nammayatri/images/user/ny_ic_information_grey.png", text : (getString ABOUT), tag : SETTINGS_ABOUT} push
+      else settingsMenuView {imageUrl : "ny_ic_emergency_contacts,https://assets.juspay.in/nammayatri/images/user/ny_ic_emergency_contacts.png" , text : (getString EMERGENCY_CONTACTS)  , tag : SETTINGS_EMERGENCY_CONTACTS, iconUrl : ""} push
+    , settingsMenuView {imageUrl : "ic_help,https://assets.juspay.in/nammayatri/images/user/ic_help.png", text : (getString HELP_AND_SUPPORT), tag : SETTINGS_HELP, iconUrl : ""} push
+    , settingsMenuView {imageUrl : "ic_change_language,https://assets.juspay.in/nammayatri/images/user/ic_change_language.png", text : (getString LANGUAGE), tag : SETTINGS_LANGUAGE, iconUrl : ""} push
+    , linearLayout
+      [ width MATCH_PARENT
+      , height (V 1)
+      , background Color.grey900
+      , margin ( MarginVertical 8 8 )
+      ][]
+    , settingsMenuView {imageUrl : "ic_share,https://assets.juspay.in/nammayatri/images/user/ic_share.png", text : (getString SHARE_APP), tag : SETTINGS_SHARE_APP, iconUrl : ""} push
+    , settingsMenuView {imageUrl : "ic_graph_black,https://assets.juspay.in/nammayatri/images/user/ic_graph_black.png", text : (getString LIVE_STATS_DASHBOARD), tag : SETTINGS_LIVE_DASHBOARD, iconUrl : "ic_red_icon,https://assets.juspay.in/nammayatri/images/user/ic_red_icon.png"} push
```

Comment:
can you add the version check for this feature as it should be working with older version